module/VuFindApi/config/module.config.php

```diff
@@ -18,14 +20,34 @@
     ],
     'service_manager' => [
         'factories' => [
+            'VuFindApi\ApiPluginManager' => 'VuFind\ServiceManager\AbstractPluginManagerFactory',
```

Comment:
Is this unused? I don't see a definition for this class.

Reply:
Unused, remnants of a bad idea. Now removed.